networkit/cpp/distance/test/SSSPGTest.cpp

```diff
@@ -67,8 +67,8 @@ TEST_F(SSSPGTest, testShortestPaths) {
     });
     count dist = bfs.distance(x);
     std::set<std::vector<node>> paths = bfs.getPaths(x, true);
-    count i = 0;
-    for (auto path : paths) {
+    [[maybe_unused]] count i = 0;
+    for (auto const &path : paths) {
```

Comment:
For consistency with the codebse, use west-const:
```suggestion
    for (const auto &path : paths) {
```